config.py

```diff
@@ -0,0 +1,6 @@
+
+ACCESS_TOKEN = 'TorBVyNG5-q_iBfTT-HmhyEHA6Q'
```

Comment:
Don't publish access tokens in public, let's remove this file

https://stackoverflow.com/questions/12481639/remove-files-from-git-commit